### ISSUE TITLE
List JuliaPackageComparisons on https://julialang.org/packages/

### ISSUE THREAD

**Issue by KronosTheLate** [JuliaLang/www.julialang.org#2037]
The resources listed under https://julialang.org/packages/ are great. Me and @hyrodium are currently working towards building up juliapackagecomparisons.github.io/, as a resource that focuses on comparisons of packages with overlapping functionality, written by humans, for humans, in more than a single sentence.

The content on the website is still a work in progress. But once all current pages have some minimum of content, would it be possible to get the resource listed on https://julialang.org/packages/? Just for reference, this would close [#2 in the JPC repo](https://github.com/JuliaPackageComparisons/JuliaPackageComparisons.github.io/issues/2).

**Comment by LilithHafner** [comment]
Yes! Feel free to make a PR and ping me once https://juliapackagecomparisons.github.io is ready.

**Comment by TheCedarPrince** [comment]
This is really great @KronosTheLate ! I had no idea it existed -- would contributions from Health specific applications be helpful?

**Comment by KronosTheLate** [comment]
The intention is to provide an overview of packages that overlap partly or significantly, in any domain. So yes!

I am also not finding that much time to work on the project, so more controbutors (or maintainers!) are always welcome 😁

**Comment by ViralBShah** [comment]
Closing this here for now as no action for website maintainers. Maybe announce it on discourse to see if folks want to help out?
